### ISSUE TITLE
Howto close activity from javascript?

### ISSUE THREAD

**Issue by stefankaifer** [olton/Metro-UI-CSS#1457]
Hi

sorry, but i don't know, how can I close an activity from javscript.
This code opens an activity, but the close-function is not working:

```
function openWaitingDialog(waiting) {
    var message = 'Please wait...';
    if (waiting.message && '' !== waiting.message) {
        message = waiting.message;
    }
    var activity = Metro.activity.open({
        type: 'square',
        style: 'color',
        overlayColor: '#76d7ea',
        overlayAlpha: 0.9,
        overlayClickClose: false,
        text: message
    });
    return activity;
}

function closeWaitingDialog(wDialog) {
    Metro.activity.close(wDialog);
}

var wDialog = openWaitingDialog({ message: 'Saving...' });
var oActivity = Metro.getPlugin($('.activity-square').first(), "activity");

// both not working:
closeWaitingDialog(wDialog);
closeWaitingDialog(oActivity);
```

Thanks for help.
Regards

Stefan

**Comment by olton** [comment]
Fixed in 4.3.3

**Comment by stefankaifer** [comment]
Many Thanks!
